ts/indicators/slow-stochastic.src.ts

```diff
@@ -0,0 +1,191 @@
+            // for inheritance
+        }
+
+        class SlowStochasticIndicatorPoint extends StochasticIndicatorPoint {
+            public series: SlowStochasticIndicator;
+        }
+
+        interface SlowStochasticIndicatorOptions
+            extends StochasticIndicatorOptions, MultipleLinesIndicatorOptions {
+            params?: SlowStochasticIndicatorParamsOptions;
+        }
+
+        interface SeriesTypesDictionary {
+            slowstochastic: typeof SlowStochasticIndicator;
+        }
+    }
+}
+
+import requiredIndicator from '../mixins/indicator-required.js';
+
+var seriesTypes = H.seriesTypes;
+
+/**
+ * The Slow Stochastic series type.
+ *
+ * @private
+ * @class
+ * @name Highcharts.seriesTypes.slowstochastic
+ *
+ * @augments Highcharts.Series
+ */
+H.seriesType<Highcharts.SlowStochasticIndicator>(
+    'slowstochastic',
+    'stochastic',
+    /**
+     * Slow Stochastic oscillator. This series requires the `linkedTo` option
+     * to be set and should be loaded after `stock/indicators/indicators.js`
+     * and `stock/indicators/stochastic.js` files.
+     *
+     * @sample stock/indicators/slow-stochastic
+     *         Slow Stochastic oscillator
+     *
+     * @extends      plotOptions.stochastic
+     * @since        8.0.0
+     * @product      highstock
+     * @requires     stock/indicators/indicators
+     * @requires     stock/indicators/stochastic
+     * @requires     stock/indicators/slowstochastic
+     * @optionparent plotOptions.slowstochastic
+     */
+    {
+        params: {
+            /**
+             * Periods for Slow Stochastic oscillator: [%K, %D, SMA(%D)].
+             *
+             * @type    {Array<number,number,number>}
+             * @default [14, 3, 3]
+             */
+            periods: [14, 3, 3]
+        }
+    },
+    /**
+     * @lends Highcharts.Series#
+     */
+    {
+        nameBase: 'Slow Stochastic',
+        init: function (this: Highcharts.SlowStochasticIndicator): void {
+            const args = arguments,
+                ctx = this;
+
+            requiredIndicator.isParentLoaded(
+                (H.seriesTypes.stochastic as any),
+                'stochastic',
+                ctx.type,
+                function (indicator: Highcharts.Indicator): undefined {
+                    indicator.prototype.init.apply(ctx, args);
+                    return;
+                }
+            );
+        },
+        getValues: function<TLinkedSeries extends Highcharts.Series> (
+            this: Highcharts.SlowStochasticIndicator,
+            series: TLinkedSeries,
+            params: Highcharts.SlowStochasticIndicatorParamsOptions
+        ): (Highcharts.IndicatorValuesObject<TLinkedSeries>|undefined) {
+            const periods: Array<number> = (params.periods as any),
+                fastValues = seriesTypes.stochastic.prototype.getValues.call(
+                    this,
+                    series,
+                    params
+                ) as any,
```

Comment:
Decided to cast to any, I couldn't resolve that issue. It seems that getting values from other indicators is a bit cumbersome. FYI @bre1470

Reply:
If I understand it correctly, stochastic is expecting a two-dimensional series (OHLC), but limiting the generic variable `TValueSeries` leads to inheritence conflict. So I agree with that.

Reply:
Maybe somewhere `Highcharts.IndicatorValuesObject<TLinkedSeries>` makes this incompatible?

Just tried to compare with `ppo` indicator, where it's first cast as `IndicatorValuesObject`, then casted to `any`:
```js
            LPE = EMA.prototype.getValues.call(this, series, {
                index: index,
                period: periods[1]
            }) as Highcharts.IndicatorValuesObject<TLinkedSeries>;

            ...

            for (...) {
                PPO.push([(LPE as any).xData[i], oscillator]);
                xData.push((LPE as any).xData[i]);
                yData.push(...);
            }
```

Anyway, if this makes sense to you @bre1470 then I'm fine :)